nulogy-fitter-happier.gemspec

```diff
@@ -25,12 +25,12 @@ Gem::Specification.new do |s|
   s.test_files   = Dir["spec/**/*"]
   s.require_path = 'lib'
 
-  s.add_dependency 'actionpack', '>= 6.0', '< 8.0'
-  s.add_dependency 'activerecord', '>= 6.0', '< 8.0'
+  s.add_dependency 'actionpack', '>= 7.2', '< 8.0'
+  s.add_dependency 'activerecord', '>= 7.2', '< 8.0'
   s.add_dependency 'newrelic_rpm', '>= 6.15.0', '< 10.0'
   s.add_dependency 'railties', '>= 6.0', '< 8.0'
```

Comment:
The version of `railties` should match that of `actionpack` and `activerecord` as they are always released together and match a particular version of the `rails` gem.

Reply:
smh